src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`}
+      </p>
```

Comment:
you can move this expression to a separate const

Reply:
> you can move this expression to a separate const

sorry, could you show it? i don't understand what do you mean